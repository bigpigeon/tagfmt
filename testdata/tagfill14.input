//tagfmt -f "json=or(:tag,lower_camel(:field))|pflag=or(:tag,snake(:field))|bson=or(:tag,lower_camel(:field))"

package main

type OrderDetail struct {
    ID       string   `json:"Id"`
    UserName string   `json:"User"`
}
