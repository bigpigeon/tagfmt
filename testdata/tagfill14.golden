//tagfmt -f "json=or(:tag,lower_camel(:field))|pflag=or(:tag,snake(:field))|bson=or(:tag,lower_camel(:field))"

package main

type OrderDetail struct {
	ID       string `json:"Id"   bson:"iD"       pflag:"id"`
	UserName string `json:"User" bson:"userName" pflag:"user_name"`
}
